a.js

```diff
@@ -0,0 +1,3 @@
+var bod_neme;
```

Comment:
The file has increased in 3 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._